src/main/java/org/wso2/carbon/mongodb/util/MongoDatabaseUtil.java

```diff
@@ -63,6 +55,11 @@
      * @throws UserStoreException if any error occurred
      */
     public static synchronized DB getRealmDataSource(RealmConfiguration realmConfiguration) throws UserStoreException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("getRealmDataSource(RealmConfiguration realmConfiguration)");
```

Comment:
can we change this to "Creating mongo datasource with realm configuration "+ realmConfiguration+getDescription